src/CONST.ts

```diff
@@ -5263,6 +5279,11 @@ const CONST = {
             REPORT_ID: 'reportID',
             KEYWORD: 'keyword',
         },
+        QUERY_KIND: {
```

Comment:
```suggestion
        QUERY_TYPE: {
```

Reply:
This is a controversial topic. The problem is that we already have `query type`. We also have `search type`. I would love to see something other than `type` in the context of search because it may be confusing for people. It doesn't have to be `kind`. Just something else.

Reply:
I agree with the "controversy" 😅 there is a `type` within the query already, part of filters. it's the type of report. And I can image people will call it `query type` just like they would might say `query status`.
Then we have the `type` of search results, which can be `transaction` or `report`.

It would nice to not name this thing also `type`, future devs will get confused.

Reply:
I like this idea! Let's go with `isCustomQuery`